src/page-modules/assistant/client/journey-planner/index.ts

```diff
@@ -0,0 +1,28 @@
+import { TripData } from '@atb/page-modules/assistant/server/journey-planner/validators';
+import { TripQuery } from '@atb/page-modules/assistant/';
```

Comment:
I'm being somewhat particular on these imports now, but I try to follow rules as:

- Locally inside a module: Relative paths.
- Usage from outside module: Absolute path

This is to indicate coupling of files and separate between external files and internal files.